src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommenderTest.java

```diff
@@ -15,7 +15,16 @@
     public void testDataInfo() throws IOException, TasteException {
         //download movies.txt.gz from 
         //    http://snap.stanford.edu/data/web-Movies.html
-        MovieRecommender recommender = new MovieRecommender("/path/to/movies.txt.gz");
+
```

Comment:
Avoid adding unused code on the PR's. It's ok to have code commented in your local changes, but if this code is not going to be used is better to get rid of it once the task is done. This kind of code increases the noise on the files and makes them hard to review.

Reply:
Nice! that makes sense, I thought that it'll be useful for the reviewers to know what's going on, but they already know haha, thanks Iván!